code/datums/exonet.dm

```diff
@@ -213,7 +219,7 @@ GLOBAL_LIST_EMPTY(exonets)
 		user.StoreMemory("Your email account address has been changed to [new_login].", /decl/memory_options/system)
 	if(issilicon(user))
 		var/mob/living/silicon/S = user
-		var/datum/nano_module/email_client/my_client = S.get_subsystem_from_path(/datum/nano_module/email_client)
+		var/datum/nano_module/program/email_client/my_client = S.get_subsystem_from_path(/datum/nano_module/program/email_client)
```

Comment:
making it a /program module makes it impossible for silicons to use, since it requires a NTOS device to run.

Reply:
Not a note for you specifically,  but that really feels like something important that should be in a comment or document somewhere in /program, because I had no idea and it has now wasted a significant chunk of time I spent fixing what I thought was outdated code.